src/test/java/starter/ReportSender.java

```diff
@@ -0,0 +1,50 @@
+package starter;
+
+import integrations.testrail.APIException;
+import integrations.testrail.TestRailManager;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import org.testng.ITestResult;
+import org.testng.annotations.AfterMethod;
+import utils.ContextKeeper;
+
+import java.io.IOException;
+
+import static integrations.SlackIntegration.sendMessageToSlack;
+
+public class ReportSender {
+
+    protected String testCaseId;
+
+    @Before
+    public static void setPreconditions() {
+
+    }
+
+    @After
+    public static void sendSerenityReportToSlack() {
+        sendMessageToSlack("Serenity Report: file:///C:/Mentoring/project/Serenity_Framework/target/site/serenity/index.html");
+    }
+
+    @After
+    public static void tearDownForTestRail() {
+        String testId = String.valueOf(ContextKeeper.INSTANCE.getTestCaseId());
+        try {
+            TestRailManager.addResultForTestCase(testId, TestRailManager.TEST_CASE_PASSED_STATUS, "");
+        } catch (IOException e) {
+            e.printStackTrace();
+        } catch (APIException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @AfterMethod
```

Comment:
to implement the same with Cucumber please consider
_@After
public void doSomethingAfter(Scenario scenario){
    // Do something after after scenario
}_